api/modules/automations.js

```diff
@@ -20,6 +20,20 @@ const automations = module.exports = (() => {
         }
     }
 
+    const deleteDraftInvoicesFromStripe = (params) => {
+        let deletedInvoice
+        try {
+            deletedInvoice = db.models.Payment.destroy({
+                where: {
+                    external_uuid: params.invoiceId
+                }
+            })
+        } catch (err) {
+            console.log(`An error ocurred: ${err}`)
+        }
```

Comment:
Let's tweak this to address some issues:

- If the payment does exist but it failed to delete for some reason, we want to send back a status of 400 (check what Stripe expects in webhook responses) to make sure Stripe tries again when possible
- If the payment does not exist, then we need to make sure Stripe does not keep trying

Reply:
If we send another code different than 2xx, stripe will keep trying several times until it stops and marks the event as failure.
Changing so we send a status of 400 in case of failure.
More details on https://stripe.com/docs/webhooks/build